### PR TITLE
Content/guidance key - Paragraph

### PR DESCRIPTION
**Pull request by andij**

### CODE REVIEW

File: src/content/docs/components/ns-card.mdx [hunk 1]
```diff
@@ -76,7 +76,7 @@ Cards are to be used to apply a container around a related grouping of informati
 | Key | Field type | Guidelines |
 | :--- | :--- | :--- |
 | <span class="marker">A</span> | Heading | The recommended length is between 4 and 12 words, not exceeding 50 characters in total. Headings contain light and bold font weight to highlight key messaging. |
-| <span class="marker">B</span> | Content | A single, short paragraph works best. You are able to use inline text links within the paragraph if necessary. Don't add additional headings within the paragraph and the copy length should not exceed more than two paragraphs of 3 lines each. Can contain bold copy `<b>`, inline links `<a>`, lists `<ul>` `<ol>`, and a caveat at the end of the relevant paragraph if required `<a href="#caveat">1</a>`. |
+| <span class="marker">B</span> | Paragraph | A single, short paragraph works best. Copy length should not exceed two paragraphs of 3 lines. Include emphasis with `<strong>`, inline links `<a>`, lists `<ul>` `<ol>`. Add [caveat link](/components/ns-caveat) if required. |
```
Comment (RobTobias123) [inline src/content/docs/components/ns-card.mdx:79]:
```suggestion
| <span class="marker">B</span> | Paragraph | A single, short paragraph works best. The text length should not exceed two paragraphs of 3 lines. Include emphasis with `<strong>`, inline links `<a>`, lists `<ul>` `<ol>`. Add a [caveat link](/components/ns-caveat) if required. |
```

Comment (RobTobias123) [reply src/content/docs/components/ns-card.mdx:79]: The word 'Copy' can be ambiguous in some cases. Perhaps the word Text is more explicit? 

Also maybe add the article 'a' before 'caveat link' or alternatively make plural - 'caveat links'?

**Review by RobTobias123 — COMMENTED** [review]
Just a comment about word choice.

File: src/content/docs/components/ns-card.mdx (hunk 1 above)
Comment (andij) [reply src/content/docs/components/ns-card.mdx]: I've applied these changes.

**Review by RobTobias123 — APPROVED** [review]
Looks good - but it appears you have a merge conflict to resolve first?

**Pull request merged**